command/agent/config.go

```diff
@@ -342,6 +342,9 @@ type Config struct {
 	// they are configured with TranslateWanAddrs set to true.
 	TaggedAddresses map[string]string
 
+	// Node metadata
```

Comment:
I'd expand this comment to explain why we elide this from the JSON (since it might be stale). This will be confusing later as to why we did this :-)